end-to-end-tests/tst/helpers/base_test.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+import boto3
+import random
+import os
+import subprocess
+from resources import Resources
+
+
+class BaseTest(unittest.TestCase):
+    def setUp(self):
+        self.stack_name = "sam-temp-stack" + str(random.randint(0, 10000))
+        self.cloud_formation_client = boto3.client("cloudformation")
+
+    def make_stack(self, capabilties, template_path):
```

Comment:
SHouldn't we raise an exception if this fails

Reply:
Updated the code with exceptions where ever applicable